src/assets/scss/modules/index.scss

```diff
@@ -1,4 +1,3 @@
-@import 'bulma';
```

Comment:
Hmm.. then we have to find another way because if we import it like this, it will be included in each module separately (thats why bundle size was so big)